Description: enabling IPv6 is easy!
Author: A. Schulze
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: smf-spf-2.0.2/smf-spf.c
===================================================================
--- smf-spf-2.0.2.orig/smf-spf.c	2012-09-07 10:19:46.000000000 +0200
+++ smf-spf-2.0.2/smf-spf.c	2012-09-07 10:19:48.000000000 +0200
@@ -716,6 +716,7 @@
     SPF_server_set_rec_dom(spf_server, context->site);
     if (!(spf_request = SPF_request_new(spf_server))) goto done;
     SPF_request_set_ipv4_str(spf_request, context->addr);
+    SPF_request_set_ipv6_str(spf_request, context->addr);
     SPF_request_set_helo_dom(spf_request, context->helo);
     SPF_request_set_env_from(spf_request, context->sender);
     if (SPF_request_query_mailfrom(spf_request, &spf_response)) {
